plugins/trendmicro_visionone/plugin.spec.yaml

```diff
@@ -1672,4 +1982,424 @@ actions:
         description: The details of the alert
         required: true
         example: { }
+  get_custom_script_list:
+    title: Get Custom Script List
+    description: Retrieves information about the available custom scripts and displays the information in a paginated list
+    input:
+      query_op:
+        title: Query Operator
+        type: string
+        description: Logical operator to employ in the query. (AND/OR)
+        required: true
+        example: " or "
+        default: " or "
+        enum:
+          - " or "
+          - " and "
+      fields:
+        title: Fields
+        type: object
+        description: JSON object of fields to query by fileName or fileType
+        required: true
+        example: { }
+    output:
+      custom_scripts_list_resp:
+        title: Custom Scripts List Response
+        description: Custom Scripts List Response Array
+        type: "[]custom_scripts_list_resp"
+        required: true
+        example: [ ]
```

Comment:
If that'd be possible please fill all the examples from `help.md` file to `plugin.spec.yaml` file.